src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/CourseActivityBuilder.kt

```diff
@@ -25,7 +24,7 @@ class CourseActivityBuilderFactory(
     endDate: String? = null,
     minimumIncentiveLevelCode: String = "STD",
     internalLocationId: Long? = -8,
-    payRates: List<CourseActivityPayRateBuilder> = listOf(courseActivityPayRateBuilderFactory.builder()),
+    payRates: List<CourseActivityPayRateBuilder> = listOf(),
```

Comment:
Defaults are now done in the DSL interface. Currently removed the default pay rate entirely because with multiple child entities you add more data rather than replace existing data - so start empty and explicitly add children.